src/utils/eth_transaction.cairo

```diff
@@ -33,6 +33,7 @@ namespace EthTransaction {
         v: felt,
         r: Uint256,
         s: Uint256,
+        nonce : felt,
```

Comment:
can we keep the same order of returned values as in the tx definition? (see link below)

```
@dataclass
class TransactionLegacy:
	signer_nonce: int = 0
	gas_price: int = 0
	gas_limit: int = 0
	destination: int = 0
	amount: int = 0
	payload: bytes = bytes()
	v: int = 0
	r: int = 0
	s: int = 0
```